pkg/jobs/hive/hive.go

```diff
@@ -46,21 +40,16 @@ func ActivateDeploy(hiveset hiveclient.Interface, clusterName string) error {
 	}
 
 	klog.V(2).Info("Found cluster " + cluster.Name + " ✓")
-	if cluster.Spec.InstallAttemptsLimit == nil || *cluster.Spec.InstallAttemptsLimit != 0 {
-		return errors.New("ClusterDeployment.spec.installAttemptsLimit is not 0")
-	}
-
-	// Update the installAttemptsLimit
-	intValue := int32(1)
-	patch := []patchStringValue{{
-		Op:    "replace",
-		Path:  "/spec/installAttemptsLimit",
-		Value: intValue,
-	}}
-	patchInBytes, _ := json.Marshal(patch)
-	*cluster.Spec.InstallAttemptsLimit = 1
-	_, err = hiveset.HiveV1().ClusterDeployments(clusterName).Patch(
-		context.TODO(), clusterName, types.JSONPatchType, patchInBytes, v1.PatchOptions{})
+	annotations := cluster.GetAnnotations()
+	if annotations[hiveconstants.ReconcilePauseAnnotation] != "true" {
+		return errors.New("clusterDeployment should have true pause annotation")
```

Comment:
If the curator only has posthook, should we still need this annotation?

Reply:
no, we also need to remove the annotation, otherwise the hive will not provision the cluster.